DataFormats/TrackReco/src/classes_def.xml

```diff
@@ -335,8 +336,9 @@
   <class name="edm::Ref<std::vector<reco::TrackExtra>,reco::TrackExtra,edm::refhelper::FindUsingAdvance<std::vector<reco::TrackExtra>,reco::TrackExtra> >"/>
   <class name="edm::RefVector<std::vector<reco::TrackExtra>,reco::TrackExtra,edm::refhelper::FindUsingAdvance<std::vector<reco::TrackExtra>,reco::TrackExtra> >"/>
 
-  <class name="reco::Track" ClassVersion="20">
-   <version ClassVersion="20" checksum="2864582648"/>
+  <class name="reco::Track" ClassVersion="21">
+   <version ClassVersion="21" checksum="1132360123"/>
```

Comment:
I think this needs to be coherent w/ the above line, and the `checksum` for the `ClassVersion` 20 and 21 should be different, I guess

Reply:
Yes, but if we change it with reservedForUpgrade1 we avoid this problem